addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -165,20 +164,16 @@ export class Base {
 
         return serializedData;
     }
-    get raw() {
-        if (this.id in this._raw) {
```

Comment:
Is the `this.id in this._raw` O(n) instead of O(1)?

Reply:
if is this._raw is an array is O(n)